src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,65 @@
+package storage;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import tasks.Task;
+import tasks.TaskList;
+
+/**
+ * The Storage class is responsible for reading and writing tasks to the file.
+ */
+public class Storage {
+    private final String filePath;
```

Comment:
Would be better if we use FILE_PATH here instead?

Reply:
It would indeed I shall fix that.